spec/views/groups/groups_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Groups', type: :feature do
+  describe 'GET groups#index' do
+    before(:each) do
+      user = User.create name: 'anonymus', email: 'anonymus@anon.co', password: 'password'
+
+      visit new_user_session_path
+      page.fill_in placeholder: 'Email', with: user.email
+      page.fill_in placeholder: 'password', with: user.password
+      click_button 'Save'
```

Comment:
- [ ] Two of the view specs happen to be failing. Kindly ensure that **all** test cases are passing.

**Failing test cases:**
![image](https://user-images.githubusercontent.com/12745474/204085847-3cf18b81-2374-4ef1-ab54-200052d73158.png)